include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -752,6 +752,32 @@ namespace VectorTools
       return true;
     }
 
+    template <typename number>
+    void invert_mass_matrix(const SparseMatrix<number> &mass_matrix,
+                            const Vector<number> &rhs,
+                            Vector<number> &solution)
```

Comment:
Ah, yes, I suppose it should be `rhs`. I had implicitly assumed that the function has the same interface as `Solve*::solve()`, where the second argument is the solution and the third the rhs. But this works as well.